apps/api/src/app/widgets/e2e/get-unseen-count.e2e.ts

```diff
@@ -155,12 +179,12 @@ describe('Unseen Count - GET /widget/notifications/unseen', function () {
       }
     );
 
-    seenCount = (await getFeedCount({ seen: false })).data.count;
+    seenCount = (await getUnseenCount({ seen: false })).data.count;
     expect(seenCount).to.equal(2);
   });
 
-  async function getFeedCount(query = {}) {
-    const response = await axios.get(`http://localhost:${process.env.PORT}/v1/widgets/notifications/count`, {
+  async function getUnseenCount(query = {}) {
+    const response = await axios.get(`http://localhost:${process.env.PORT}/v1/widgets/notifications/unseen`, {
```

Comment:
we had count tests instead of unseen count, those tests should have been under get-count e2e. i moved them there and created proper tests for unseen count one.